cms/src/main/java/uk/nhs/digital/apispecs/jcr/JcrDocumentLifecycleSupport.java

```diff
@@ -61,13 +61,32 @@ public void setProperty(final String propertyName, final String value) {
             .flatMap(node -> JcrNodeUtils.getStringPropertyQuietly(node, propertyName));
     }
 
+    public Optional<Instant> getInstantProperty(final String propertyName,
```

Comment:
Hmm, I see your point but I think I'll leave it - and other getters as they are.

The reason is that no checkout is required for reading values and so no getter triggers checkout or sets 'dirty' flag, here. For this reason, I don't think we need to emphasise the 'NoCheckout' aspect on getters because it's the only mode of reading properties.